pkg/crc/segment/segment.go

```diff
@@ -140,7 +140,8 @@ func readIdentifyHash(identifyHashPath string) (uint64, error) {
 	}
 
 	if length := len(cachedHashBytes); length != 8 {
-		return 0, fmt.Errorf("Got %d bytes hash, expected 8", length)
+		logging.Warnf("Got %d bytes hash (expected 8), identify hash file will be regenerated", length)
```

Comment:
Does the actual generation be logged too? If so, the text would be better to be written as:

```patch
+     logging.Warnf("Got %d bytes hash (expected 8), identify hash file will have to be regenerated", length)
```

as the function itself will not do this, but might imply this when you read the log.

Reply:
>  logging.Warnf("Got %d bytes hash (expected 8), identify hash file will have to be regenerated", length)

I did not want to use this wording as it would (in my opinion) imply that an action from the user is needed. The file is generated automatically as soon as possible by other functions.